### PR TITLE
Customize tab badge color

### PR DESCRIPTION
**Pull request by bernabe9**
# Customize tab badge color

### Before:(default color)
screen shot 2017-09-26 at 1 38 46 pm

### After:(with orange color)
screen shot 2017-09-26 at 2 41 30 pm

### CODE REVIEW

**Comment by henrikra** [comment]
Clean and simple 👍

**Comment by bernabe9** [comment]
@gran33 Yes, the default color is the same used before.

**Comment by gran33** [comment]
@bernabe9 in order to get this PR, can u plz see my reviews comments I left?

**Comment by bernabe9** [comment]
@gran33 I already answered your comment about the default color... I don't see any other comments.

**Comment by bernabe9** [comment]
@gran33 fixed, sorry for the delay

File: ios/RCCTabBarController.m [hunk 1]
```diff
@@ -263,6 +263,9 @@ - (void)performAction:(NSString*)performAction actionParams:(NSDictionary*)actio
       }
       else
       {
+        NSString *badgeColor = actionParams[@"badgeColor"];
+        UIColor *color = badgeColor != (id)[NSNull null] ? [RCTConvert UIColor:badgeColor] : nil;
```
Comment (gran33) [inline ios/RCCTabBarController.m]: 🙏

**Comment by gran33** [comment]
@bernabe9 Great job, Thanks! 😍

**Comment by gran33** [comment]
A small fix that will lead to crash:
https://github.com/wix/react-native-navigation/pull/2068

**Pull request merged**